loadtest/user/userentity/actions.go

```diff
@@ -82,6 +82,19 @@ func (ue *UserEntity) CreateGroupChannel(memberIds []string) (string, error) {
 	return channel.Id, err
 }
 
+func (ue *UserEntity) CreateDirectChannel(userId1, userId2 string) (*model.Channel, error) {
```

Comment:
I think we can simplify. One of the two users is going to be the current user so we just need the id of the other and leverage the store to get the current user id.

Also let's not return a pointer to the channel but just the channel id (we don't want to break immutability).